composer.json

```diff
@@ -1,65 +1,74 @@
 {
-    "name": "drupal-composer/drupal-project",
-    "description": "Project template for Drupal 8 projects with composer",
-    "type": "project",
-    "license": "GPL-2.0+",
-    "authors": [
-        {
-            "name": "",
-            "role": ""
-        }
-    ],
-    "repositories": [
-        {
-            "type": "composer",
-            "url": "https://packagist.drupal-composer.org"
-        }
+  "name": "drupal-composer/drupal-project",
```

Comment:
AFAIK D.O packagist still have some rough edges. I'll try to move forward an discuss about packagist in another issue